amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2033,3 +2030,70 @@ vx_status releaseGraphHandle(vx_node node, RPPCommonHandle *handle)
     return VX_SUCCESS;
 }
 #endif
+
+void fillDescriptionPtrfromDims(RpptDescPtr &descPtr, Rpp32s layout, size_t *tensorDims) {
+    switch(layout) {
+        case 0: { // For NHWC
+            descPtr->n = tensorDims[0];
+            descPtr->h = tensorDims[1];
+            descPtr->w = tensorDims[2];
+            descPtr->c = tensorDims[3];
+            descPtr->strides.nStride = descPtr->c * descPtr->w * descPtr->h;
+            descPtr->strides.hStride = descPtr->c * descPtr->w;
+            descPtr->strides.wStride = descPtr->c;
+            descPtr->strides.cStride = 1;
```

Comment:
@fiona-gladwin shouldn't this stride be sizeof(data_type) and similarly the rest multiplied with the same value?
for ex: with vx_tensor of type float32, stride[0] = sizeof(float32), stride[1] = input_dims[0]*sizeof(float32) etc

Reply:
Currently we are modifying the stride value in rocAL to match with stride calculations in RPP.